File: Dante1/Module1.02/Zadanie2.06/main.c
#include <stdio.h>

int main()
{
    int num1, num2;

    printf("Enter num1: ");

    int check1 = scanf("%i",&num1);
    if(check1 != 1)
    {
        printf("Incorrect input");
        return 1;
    }

    printf("Enter num2: ");

    int check2 = scanf("%i",&num2);
    if(check2 != 1)
    {
        printf("Incorrect input");
        return 1;
    }

    if(num2 == 0)
    {
        printf("Operation not permitted");
        return 1;
    }
    else if(num1 % num2 == 0)
    {
        printf("%i is divisible by %i",num1,num2);
    }
    else
    {
        printf("%i is not divisible by %i",num1,num2);
    }
    return 0;
}

File: Dante1/Module1.02/Zadanie2.06/main.rs
fn main()
{
    println!("Enter first number: ");
    let mut number_one :String = String::new();
    std::io::stdin().read_line(&mut number_one).expect("Incorrect input in String for number one");
    
    let number_one: i32 = number_one.trim().parse().expect("Error in parsing number_one");
    
    println!("Enter number two:");
    let mut number_two: String = String::new();
    std::io::stdin().read_line(&mut number_two).expect("Incorrect input in String for number two");
    
    let number_two: i32 = number_two.trim().parse().expect("Error in parsing number_two");
    if number_two == 0
    {
        panic!("Error! Division by 0!");
    }
    
    
    if number_one % number_two == 0 && number_one > number_two
    {
        println!("{} is divisible by {}", number_one, number_two);
    }
    else 
    { 
        println!("{} is not divisible by {}", number_one, number_two);
    }
    
    return;
}
